ros_pre-release.sh

```diff
@@ -71,9 +72,10 @@ travis_time_start run_prerelease
 travis_time_end  # run_prerelease
 
 travis_time_start show_testresult
-catkin_test_results --verbose && (echo 'ROS Prerelease Test went successful.'; exit 0) || error
+catkin_test_results --verbose && { echo 'ROS Prerelease Test went successful.'; RETVAL=0; success; } || { RETVAL=1; error; }
 travis_time_end  # show_testresult
```

Comment:
So `travis_time_end` should not be used if block is terminated by `success` and `error`?